doc/api/cli.md

```diff
@@ -53,6 +53,11 @@ changes:
 Evaluate the following argument as JavaScript. The modules which are
 predefined in the REPL can also be used in `script`.
 
+*Note*: Please use double quote for the `script`, although it does not matter
```

Comment:
@lukaszewczak Thank you for posting this.
I would suggest the following change:
```markdown
*Note*: On Windows, using `cmd.exe` a single quote will not work correctly
because it only recognizes double `"` for quoting. In Powershell or
Git bash, both `'` and `"` are usable.
```
1. Start with `On Windows` so it's clear that the note target Windows behaviur
2. IMHO there is no need to compare to Linux (also the comparison is with Linux'es shells, the OS does care)
3. Mention that it's only `cmd.exe`, as PS and bash on windows understand `'`

Reply:
Thank you @refack for your comment. I made a change in note according to your suggestions.